src/main/kotlin/no/nav/familie/ef/iverksett/lagreIverksett/tjeneste/LagreIverksettService.kt

```diff
@@ -0,0 +1,17 @@
+package no.nav.familie.ef.iverksett.lagreIverksett.tjeneste
+
+import arrow.core.Either
```

Comment:
Usikker på om jeg tror vi trenger Either her? 

For http-kall som kommer inn og feiler kan vi håndtere dette med å kaste kontrollerte feil samt bruke en `ExceptionHandler` som logger i securelogger samt returnerer `500-feil`

Reply:
Ja, du tenker på et advice ? Vi kunne lagt på advice i controlleren, og så brukt Either ellers, sånn at vi slipper å kaste unntak ?

Reply:
Men hva ønsker vi å få ut av Either?  

- Task-rammeverket forholder seg til at ting går bra eller dårlig (exceptions)  ? 
- Controlleren forholder seg til enkle kall som skal gå bra eller dårlig ? 

Jeg finner ikke et foreløpig bruksområde for Either enn så lenge..?

Reply:
Tja, tenkte det var greit å deklarere feiltyper på det viset, hvis man har lyst til å forholde seg til det noe sted. Men vi kan jo evt bare la unntak som skulle oppstå alltid gå helt gjennom til controlleren, hvis det er det du tenker ?

Reply:
Jeg er ikke vant til å lese Either, så jeg er på gjerdet her. Akkurat nå er jeg litt usikker.

Reply:
Enig i at det er mer beskrivende feil - og det er veldig nyttig i situasjoner der man skal utføre X hvis feilen er 1, Y hvis feilen er 2, Z hvis feilen er 3 osv. 

Det kan også være veldig nyttig med Either hvis man godtar at f.eks 2 av 5 kall kan feile og allikevel fullføre operasjonen. F.eks uthenting av personopplysninger i sak; MEDL (kan godta at feiler), PDL (godtar IKKE at feiler), NAV-kontor (kan godta at feiler). 

Derimot er jeg usikker på bruk av Either i ren "CRUD" - og slik vi bruker task-rammeverket så er dette et forsøk på å isolere én og én funksjon til å kjøre/rekjøre og utføre neste operasjon i kjeden.

Reply:
Ja, det kan virke litt overflødig i dette tilfellet, så jeg fjerner Either for lagring av "iverksett".